.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Remove all, we'll include only the necessary folders via !
+*
+
+!sunshine
+!cmake
+!assets
```

Comment:
This is now `src_assets`, if going with the include route, could include `src_assets/common` & `src_assets/linux`

If going the exclude route, could exclude these directories:
- `sunshine/platform/macos`
- `sunshine/platform/windows`
- `third-party/ViGEmClient`
- `src_assets/macos`
- `src_assets/windows`